src/rules/rename-default-import.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileoverview Rule to enforce aliases for default imports
+ * @author Michał Kołodziejski
+ */
+
+import docsUrl from '../docsUrl'
+
+
+function isDefaultImport(specifier) {
+  if (specifier.type === 'ImportDefaultSpecifier') {
+    return true
+  } else if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'default') {
+    return true
+  }
+  return false
+}
+
+function isCommonJSImport(declaration) {
+  const variableInit = declaration.init
+  if (variableInit.type === 'CallExpression') {
+    return variableInit.callee.name === 'require'
+  }
+  return false
+}
+
+function handleImport(context, node, specifierOrDeclaration, packageName, importAlias) {
+  const mappings = context.options[0] || {}
+  const expectedAlias = mappings[packageName]
+
+  if (expectedAlias === undefined || expectedAlias === importAlias) {
```

Comment:
when would it be `undefined`? is this prevented by a schema error, such that this should really be `has(mappings, packageName)`?

Reply:
It will be undefined if there is not specific mapping for a given package. For example, I might want to rename `prop-types` but there are other package that i don't care about.

Reply:
right - but i mean, if someone puts `{ "foo": undefined }` in their schema?

Reply:
I could always try to validate the options so that we don't have accept something like `undefined` which doesn't make sense.